--- orig
+++ stripped
@@ -28,9 +28,9 @@
 
 Atenção também para a **diferença** entre as exceções dos princípios da **exclusividade** e **anualidade**.
 
-![[Exceções ao princípio da exclusividade.png|1349]]
+!1349
 
-![[Exceções ao princípio da anualidade.png|1349]]
+!1349
 
 # **4. PRINCÍPIO DO ORÇAMENTO BRUTO**
 
@@ -45,7 +45,7 @@
 
 Veja as exceções do princípio da **<span style="color:#83a598">especificação</span>**:
 
-![[Exceções ao princípio da especificação.png|1349]]
+!1349
 
 ⚠️Programas especiais de trabalho são despesas de capital de investimentos
 
@@ -70,7 +70,7 @@
 1. **Autorização para abertura de créditos suplementares.** 
 2. **Autorização contratação de operações de crédito (INCLUSIVE ARO)**
 
-![[Exceções ao princípio da exclusividade-1.png|1349]]
+!1349
 
 ⚠️Atente-se ao fato que os créditos especiais e extraordinários **<span style="color:#fb4934">não são exceções</span>** ao princípio da exclusividade.
 
@@ -104,7 +104,7 @@
 GA - Garantia às ARO
 GA - Garantia ou contragarantia à União
 
-![[Princípio da não vinculação.png|1349]]
+!1349
 # **8. PRINCÍPIO DA PROIBIÇÃO DO ESTORNO**
 
 Outro princípio importante que você deve saber é o da **<span style="color:#d3869b">Proibição do Estorno</span>**, o qual está na CF88:
