--- orig
+++ stripped
@@ -111,7 +111,7 @@
 
 - _Art. 166, § 2º As emendas serão_ _apresentadas na Comissão mista_, que sobre elas emitirá parecer, e apreciadas_, na forma regimental, pelo_ _Plenário das duas Casas_ _do Congresso Nacional._
 
-![[Emendas.png|740]]
+!740
 
 Agora fique ligado na parte mais importante sobre as emendas, as condições para aprovação de emendas ao PLOA e ao PLDO
 
@@ -137,7 +137,7 @@
 
 As condições para Emendas ao projeto da LOA são as seguintes:
 
-![[Emendas ao projeot da loa.png|1038]]
+!1038
 
 Além disso, o Art. 33 da Lei 4.320/64 estabelece as emendas proibidas do projeto da LOA.
 
@@ -203,7 +203,7 @@
 
 Você ja sabe que o nosso orçamento é autorizativo com traços de impositivo. Esses traços estão nas emendas, sendo três tipos de emendas. Dentre as emendas coletivas, somente as emendas de bancada é que são impositivas! As emendas de comissão não o são!
 
-![[Emendas ao orçamento.png|1268]]
+!1268
 
 ⚠️Alteração recente na CF88:
 
@@ -236,7 +236,7 @@
 
 🎯Primeira coisa para você memorizar é que esse artigo aborda a transferência **somente para emendas individuais.**
 
-![[Emendas individuais impositivas.png|1513]]
+!1513
 
 - Os recursos transferidos <u><span style="color:#83a598">não</span></u> <span style="color:#83a598">integrarão a receita</span> dos entes beneficiários para fins de:
 	- Repartição 
